src/test/scala/edu/ie3/mobsim/MobilitySimulatorTestData.scala

```diff
@@ -0,0 +1,116 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.mobsim
+
+import akka.actor.ActorRef
+import edu.ie3.mobsim.io.geodata.PoiEnums.PoiTypeDictionary.{
+  CHARGING_HUB_HIGHWAY,
+  CHARGING_HUB_TOWN,
+  WORK
+}
+import edu.ie3.mobsim.model.{ChargingBehaviorTestData, ElectricVehicle}
+import edu.ie3.simona.api.data.ev.ExtEvData
+import edu.ie3.simona.api.data.ev.ontology.builder.EvMovementsMessageBuilder
+
+import java.util.UUID
+import scala.collection.immutable.{SortedSet, TreeSet}
+
+trait MobilitySimulatorTestData extends ChargingBehaviorTestData {
+  val evData: ExtEvData = new ExtEvData(ActorRef.noSender, ActorRef.noSender)
+  val builder = new EvMovementsMessageBuilder
+
+  def electricVehicles(seq: Seq[ElectricVehicle]): SortedSet[ElectricVehicle] =
+    SortedSet.empty[ElectricVehicle] ++ seq.toSet
```

Comment:
You can replace this line with `SortedSet(seq)`, which also makes this method obsolete. See below